app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -32,9 +34,8 @@ import co.nimblehq.compose.crypto.ui.userReadableMessage
 
 @Composable
 fun HomeScreen(
-    viewModel: HomeViewModel,
-    // TODO: Remove it and handle in VM instead
-    onMyCoinsItemClick: () -> Unit
+    navController: NavController,
```

Comment:
FYI: I think we should not pass NavController into composable. Let me update it in my PR 👌